test/fuzz/classifier-spdx.c

```diff
@@ -0,0 +1,76 @@
+#include "../licences.h"
+
+#include <stdio.h>
+#include <stdlib.h>
+
+const size_t CHUNK_SIZE = 4096;
+
+struct buffer {
```

Comment:
This looks like a buffer that will re-allocate memory so that whatever data was passed into it will be stored in a single chunk. This functionality is already provided by `struct ReBuffer` found in `src/buffers.h`.

Reply:
Nice, thanks!